test/src/test/java/org/corfudb/runtime/object/transactions/OptimisticTransactionContextTest.java

```diff
@@ -123,9 +125,14 @@ public void customConflictObjectsConflictAborts()
      */
     @Test
     public void checkSlowWriterTxAbortsOnHoleFill() {
+        final Duration holeFillTimeout = Duration.ofSeconds(1);
```

Comment:
@StrikingThirteen The new holeFillTimeout introduced in a previous pr has 10 secs timeout by default, which is causing this test to fail because of timeout. I agree there should some comment here or before creating`rtReader` as a reminder.

Reply:
Done.